blog/articles/get-started-knative-eventing.md

```diff
@@ -0,0 +1,93 @@
+| Title  | LinkTitle | Date | Description | Type | Aliases
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
+| Getting started with Knative Eventing using Bitcoin transaction data  | Get Started with Knative Eventing  | 2020-05-01  | Guided walkthrough on using Knative Eventing  | Blog  | /blog/2010-05-01-getting-started-knative-evetning  |
+
+I’ve been exploring [Knative Eventing](https://knative.dev/docs/eventing/), a system that enables a cloud native eventing ecosystem to be easily deployed through the use of **event producers** and **event consumers.**
+
+As part of this I put together a simple demo that someone can use to explore, what I think are, some of the key advantages of event-driven architectures, such as:
+
+- Push-based messaging.
+- True decoupling of producers and consumers.
+- Apply business logic while data is in motion.
+- Real time event-streams for data science — millisecond decisions e.g fraud detection.
+
+In this post, I’ll show you how to get an example eventing scenario up and running using some of the basic Knative eventing components; **broker, trigger, producer** and **consumer.** This demo shows streaming events in real-time, in-stream transformation and push-based front-ends in action and they can serve as a basis for you to explore further. You can use this as a basis and build out, exploring further with what is possible.
+
+This scenario uses bitcoin transaction events as an example of an event stream. The events will be displayed in real-time via and UI front-end and also classified into sizes by another service based on their total transaction value.
+
+In the diagram below, you can see a plan of what we will deploy.
+
+![Diagram of this example Knative Eventing scenario](/blog/images/knative-eventing-scenario.png)
+
+There is a github repository to accompany this demo [here](https://github.com/josiemundi/knative-eventing-blockchain-demo). The source code for all of the individual services is available in the github repo. If you want to use the pre-built images from Docker hub then you will only need the files in the yaml directory.
+
+This demo was presented at the first Knative Community Meetup and there is a video demo [here](https://www.youtube.com/watch?v=sGi_LuAaaT0).
```

Comment:
I'd pull this up higher in the document; I don't know if we want to [embed a thumbnail player of the video](https://gohugo.io/content-management/shortcodes/#youtube) or not.

Reply:
added!